rulesets/EnergyStarRuleset/resources/EnergyStarRuleset.rb

```diff
@@ -908,7 +909,10 @@ def self.set_appliances_clothes_washer_reference(orig_hpxml, new_hpxml)
       location = clothes_washer.location.gsub('unvented', 'vented')
     end
 
-    reference_values = HotWaterAndAppliances.get_clothes_washer_default_values(@eri_version)
+    if [ESConstants.SFNationalVer3_2].include? @program_version  # FIXME: Better way to handle it?
+      std_models = 'Std 2018-present'
+    end
+    reference_values = HotWaterAndAppliances.get_clothes_washer_default_values(@eri_version, std_models)
```

Comment:
I suggest not touching OS-HPXML (revert the change) and just set the clothes washer values here in this method if it's version x.2.